server/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '2'
+services:
+  pdfdraw-server:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+    environment:
```

Comment:
Could you please rename the file to `docker-compose.yml.in` and add all possible options to the `environment` block (commented out) to see what can be customized without having to check the `docker/config.js.docker`.

That way people can customize their `docker-compose.yml` file without interfering with the file in the repository (similar to what is done with the `config.js.in`). Please also add an entry to the `.gitignore` file in the repository root for `docker-compose.yml`.